mmcv/ops/csrc/common/pytorch_mlu_helper.hpp

```diff
@@ -0,0 +1,26 @@
+/*************************************************************************
```

Comment:
Some definitions at this file can be found at `mmcv/ops/csrc/common/mlu/common_mlu_helper.hpp`.
Should we keep consistent with CUDA that `mmcv/ops/csrc/common/pytorch_cuda_helper.hpp` includes the `common_cuda_helper.hpp`. https://github.com/open-mmlab/mmcv/blob/48f44b697a3510803a252c78538a9064458995a5/mmcv/ops/csrc/common/pytorch_cuda_helper.hpp#L11

@grimoire

Reply:
Yes, I think so.

Reply:
Done